Translation/en/Text/Main/CharacterMaker.txt

```diff
@@ -0,0 +1,945 @@
+//*******************************************************************
+//*******************************************************************
+// Note: This file contains only UI elements, NOT items such as
+//       clothing and accessories
+//       Items must go to RedirectedResources folder
+//*******************************************************************
+//*******************************************************************
+
+//Scope of the file: Only applies to the Chara Maker for Koikatsu Sunshine main game
+#set exe KoikatsuSunshine
+#set level 3
+
+//-------------------------------------------------------------------
+// "Universal" Text
+// For everything too generic or used in multiple places.
+//-------------------------------------------------------------------
+//Used mostly in menu buttons
+全体=General
+変形まとめ=Slider List
+
+//Exit menu
+キャラメイク終了確認=Leaving The Character Maker
+キャラメイクを終了しますか？=Quit The Character Maker?
+
+//Generic words
+はい=Yes
+する=Yes
+いいえ=No
+しない=No
+ランダム=Random
+
+//All Sliders List
+最小=Minimum
+最大=Maximum
+※ 各項目の変形に関するものをまとめたものとなります。=※ You can adjust all option sliders at once in this list.
+
+//Used for both Facepaint as well as Body Paint
+sr:"^ペイント([０-９0-9]+)の種類$"=Paint $1 Type
+sr:"^ペイント([０-９0-9]+)の色$"=Paint $1 Color
+sr:"^ペイント([０-９0-9]+)回転$"=Paint $1 Rotation
```

Comment:
For numbers I generally use `([０-９0-9①-⑨]+)` to capture, because sometimes they change things around in later releases and this catches all the options (at least until they start using the 2-digit circled numbers). Also using `sr:` for numbers introduces additional calls to the translator over just using `r:` for simple entries like this.

```suggestion
r:^ペイント([０-９0-9]+)の種類$=Paint $1 Type
r:^ペイント([０-９0-9]+)の色$=Paint $1 Color
r:^ペイント([０-９0-9]+)回転$=Paint $1 Rotation
```